lib/components/related_artists/__tests__/__snapshots__/layout-test.js.snap

```diff
@@ -0,0 +1,105 @@
+exports[`test lays out correctly 1`] = `
+<View
+  onLayout={[Function]}
+  style={
+    Object {
+      "flexDirection": "column",
+    }
+  }>
+  <Text
+    accessible={true}
+    allowFontScaling={true}
+    ellipsizeMode="tail"
+    numberOfLines={1}
+    style={
+      Array [
+        Object {
+          "fontSize": 17,
+        },
+        Object {
+          "fontSize": 20,
+        },
+        Object {
+          "fontFamily": "AGaramondPro-Regular",
+        },
+      ]
+    }>
+    Related Artists
+  </Text>
+  <View
+    style={
+      Object {
+        "alignItems": "flex-start",
+        "flexDirection": "row",
+        "flexWrap": "wrap",
+        "justifyContent": "space-around",
+        "marginLeft": -10,
+        "marginRight": -10,
+        "marginTop": 12,
+      }
+    }>
+    <TouchableWithoutFeedback
+      onPress={[Function]}>
+      <View
+        style={
+          Object {
+            "margin": 5,
+            "paddingBottom": 20,
+            "width": 242,
```

Comment:
Now we have a calculated value for width based on the mocked `nativeEvent` width.